[2022-08-06 19:37:22,870] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 19:37:22,881] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 19:37:22,881] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:22,881] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:22,881] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:22,895] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-11 00:00:00+00:00
[2022-08-06 19:37:22,900] {standard_task_runner.py:52} INFO - Started process 1362 to run task
[2022-08-06 19:37:22,904] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-11T00:00:00+00:00', '--job-id', '406', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpfkncjqc3', '--error-file', '/tmp/tmpo46ngz60']
[2022-08-06 19:37:22,904] {standard_task_runner.py:80} INFO - Job 406: Subtask second_task
[2022-08-06 19:37:22,980] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:23,085] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-11T00:00:00+00:00
[2022-08-06 19:37:23,086] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:23,086] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:23,099] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:23,101] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:23,101] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:23,142] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211211T000000, start_date=20220806T193722, end_date=20220806T193723
[2022-08-06 19:37:23,195] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:23,229] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:19,202] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 21:51:19,218] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 21:51:19,218] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:19,218] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:19,218] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:19,236] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-11 00:00:00+00:00
[2022-08-06 21:51:19,244] {standard_task_runner.py:52} INFO - Started process 1560 to run task
[2022-08-06 21:51:19,249] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-11T00:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp7un0koor', '--error-file', '/tmp/tmpm4c8wbns']
[2022-08-06 21:51:19,249] {standard_task_runner.py:80} INFO - Job 408: Subtask second_task
[2022-08-06 21:51:19,344] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:19,456] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-11T00:00:00+00:00
[2022-08-06 21:51:19,458] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:19,458] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:19,473] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:19,475] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:19,475] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:19,532] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211211T000000, start_date=20220806T215119, end_date=20220806T215119
[2022-08-06 21:51:19,580] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:19,642] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:42,687] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 21:56:42,702] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 21:56:42,702] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:42,702] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:42,702] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:42,721] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-11 00:00:00+00:00
[2022-08-06 21:56:42,728] {standard_task_runner.py:52} INFO - Started process 1355 to run task
[2022-08-06 21:56:42,731] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-11T00:00:00+00:00', '--job-id', '409', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmplfwkzav2', '--error-file', '/tmp/tmp_tdxm2gs']
[2022-08-06 21:56:42,731] {standard_task_runner.py:80} INFO - Job 409: Subtask second_task
[2022-08-06 21:56:42,807] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:42,906] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-11T00:00:00+00:00
[2022-08-06 21:56:42,907] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:42,907] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:42,919] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:42,920] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:42,921] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:42,956] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211211T000000, start_date=20220806T215642, end_date=20220806T215642
[2022-08-06 21:56:42,982] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:43,014] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:47,075] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 22:11:47,088] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 22:11:47,088] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:47,088] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:47,089] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:47,104] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-11 00:00:00+00:00
[2022-08-06 22:11:47,111] {standard_task_runner.py:52} INFO - Started process 1368 to run task
[2022-08-06 22:11:47,114] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-11T00:00:00+00:00', '--job-id', '409', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpbpdipre7', '--error-file', '/tmp/tmp9xk1895f']
[2022-08-06 22:11:47,114] {standard_task_runner.py:80} INFO - Job 409: Subtask second_task
[2022-08-06 22:11:47,201] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:47,349] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-11T00:00:00+00:00
[2022-08-06 22:11:47,350] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:47,351] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:47,370] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:47,372] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:47,372] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:47,409] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211211T000000, start_date=20220806T221147, end_date=20220806T221147
[2022-08-06 22:11:47,446] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:47,524] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:04,646] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 22:23:04,666] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [queued]>
[2022-08-06 22:23:04,666] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:04,666] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:04,667] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:04,690] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-11 00:00:00+00:00
[2022-08-06 22:23:04,698] {standard_task_runner.py:52} INFO - Started process 1412 to run task
[2022-08-06 22:23:04,703] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-11T00:00:00+00:00', '--job-id', '409', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpofdi3wal', '--error-file', '/tmp/tmpdzvv9831']
[2022-08-06 22:23:04,712] {standard_task_runner.py:80} INFO - Job 409: Subtask second_task
[2022-08-06 22:23:04,835] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-11T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:04,941] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-11T00:00:00+00:00
[2022-08-06 22:23:04,942] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:04,943] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:04,957] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:04,959] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:04,959] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:04,994] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211211T000000, start_date=20220806T222304, end_date=20220806T222304
[2022-08-06 22:23:05,038] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:05,083] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
